flow_action_components/CollectionProcessors/force-app/main/default/classes/ListActionsTest.cls

```diff
@@ -47,145 +100,218 @@ public with sharing class ListActionsTest {
         requests.add(request);
         responseWrapper = GetLookupCollection.get(requests);
         System.debug('responseWrapper is: ' + responseWrapper);
-        System.assertEquals(null, responseWrapper[0].errorText);
+        Assert.areEqual(null, responseWrapper[0].errorText);
         requests.clear();
     }
 
-    static OpportunityContactRole createOpportunityContactRole(List<Contact> testContacts, List<Opportunity> testOpportunities) {
-
-        OpportunityContactRole ocRole = new OpportunityContactRole(contactId = testContacts[0].Id, opportunityId = testOpportunities[0].Id);
+    static OpportunityContactRole createOpportunityContactRole(
+        List<Contact> testContacts,
+        List<Opportunity> testOpportunities
+    ) {
+        OpportunityContactRole ocRole = new OpportunityContactRole(
+            contactId = testContacts[0].Id,
+            opportunityId = testOpportunities[0].Id
+        );
         insert ocRole;
 
         return ocRole;
     }
 
-
     @isTest
-    static void canGetChildCollection() {
-        //user must pass in at least one input recordid or an input record
-
+    static void canGetChildCollection_inputRecord() {
```

Comment:
Previously there was a single `@isTest` method, `canGetChildCollection`, covering all `GetChildCollection` functionality. I split that up into separate methods to achieve something closer to true unit testing. Since the new `@TestSetup` method was introduced and DML is no longer performed in these tests, the performance cost should be negligible.